ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -333,9 +335,14 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 				logger.Info(fmt.Sprintf("shutdownTime not reached, requeue this RayJob for %d seconds", delta))
 				return ctrl.Result{RequeueAfter: time.Duration(delta) * time.Second}, nil
 			}
-			// We only need to delete the RayCluster. We don't need to delete the submitter Kubernetes Job so that users can still access
-			// the driver logs. In addition, a completed Kubernetes Job does not actually use any compute resources.
-			if _, err = r.deleteClusterResources(ctx, rayJobInstance); err != nil {
+			if os.Getenv(DELETE_RAYJOB_CR_AFTER_JOB_FINISHES) != "" {
```

Comment:
make the usage consistent with other feature flags.

Reply:
Done